component/events.md

```diff
@@ -50,6 +50,205 @@ Components have the ability to bind to special [can.events.inserted inserted] an
+                get: function(lastSet, deferred){
+                    this.attr("myPromise").always(deferred);
+                }
+            },
+            myPromise: {
+                Value: can.Deferred
+            }
+        }
+    });
+
+    can.Component.extend({
+        tag: "example",
+        template: '',
+        helpers: {},
+        simpleHelpers: {},
+        viewModel: ViewModel,
+        events: {
+            inserted: function(){
+                // 1.)
+                this.viewModel.attr('foo', "say hello world");
+                
+                // 2.)
+                this.viewModel.attr('myList').push({
+                    name: 'Justin'
+                });
+                
+                // 3.)
+                this.viewModel.attr('myList').splice(1,1);
+
+                // 4.)
+                this.viewModel.attr('myList', new can.List(['Maya']));
+
+                // 5.)
+                this.viewModel.attr('myList').replace(new can.List(['Peter']));
+
+                // 6.)
+                this.viewModel.attr('myMap.eggs', 10);
+
+                // 7.1.)
+                this.viewModel.attr('recipe', {name: "hot dog"});
+                this.viewModel.attr('recipe').save();
+                
+                // 8.)
+                this.viewModel.attr('myPromise', Recipe.findAll());
+            },
+            '{viewModel.myMap} change': function(){
+                console.log("viewModel's 'myMap' changed!");
+            },
+            '{viewModel} myMap.eggs': function(){
+                console.log("viewModel's 'myMap.eggs' changed!");
+            },
+            '{myMap} eggs': function(){
+                console.log("alias for '{viewModel} myMap.eggs'");
+            },
+            '{viewModel} myList': function(){
+                console.log("viewModel's 'myList' is replaced!");
+            },
+            '{viewModel.myList} change': function(){
+                console.log("viewModel's 'myList' changed!");
+            },
+            '{viewModel.myList} add': function(){
+                console.log("add a item on viewModel's 'myList'");
+            },
+            '{viewModel.myList} remove': function(){
+                console.log("remove a item on viewModel's 'myList'");
+            },
+            '{viewModel} foo': function(){
+                console.log("viewModel's 'foo' changed");
+            },
+            '{viewModel} bar': function(){
+                console.log("viewModel's 'bar' changed");
+            },
+            '{viewModel.recipe} created': function(response, event){
+                console.log("viewModel's 'recipe' created with ID: "+response.attr('id'));
+
+                // add a ID for this instance
+                this.viewModel.attr('recipe.id', response.attr('id'));
+                this.viewModel.attr('recipe.name','sushi');
+
+                // 7.2.)
+                // update
+                this.viewModel.attr('recipe').save();
```

Comment:
you create an instance, get the ID back from the server. the response is just a response, not the recipe model. 
to update the recipe model an ID have to set. if not, save() would create a new recipe